packages/reactivity/src/ref.ts

```diff
@@ -57,7 +58,7 @@ function toProxyRef<T extends object, K extends keyof T>(
       object[key] = newVal
     }
   }
-  return v as Ref<T[K]>
+  return (v as unknown) as Ref<T[K]>
```

Comment:
It would be better if

```ts
function toProxyRef<T extends object, K extends keyof T>(
  object: T,
  key: K
): Ref<T[K]> {
  return {
    [refSymbol]: true,
    get value(): any {
      return object[key]
    },
    set value(newVal) {
      object[key] = newVal
    }
  }
}
```

Reply:
Make sense